### ISSUE TITLE
unrecoverable error on startup: couldn't find monospace font

### ISSUE THREAD

**Issue by durka** [jwilm/alacritty#57]
Here is the output when starting the editor:

```
Alexs-MacBook-Air-3:alacritty alex$ target/release/alacritty
device_pixel_ratio: 1
device_pixel_ratio: 1
width: 1024, height: 768
Initializing glyph cache
Alacritty encountered an unrecoverable error:

	Couldn't find a font with name 'monospace' and style 'slant=Normal, weight=Normal'
```

A blank window briefly flashes on the screen but disappears.

OSX 10.9.5

**Comment by jwilm** [comment]
Sorry about that! We need to figure out how to have a config file that works on macOS and Linux (#40). Currently, the font lookup system for macOS isn't smart enough to find any monospace font. Alacritty should run if you replace `monospace` in your config file with `Menlo`.

**Comment by durka** [comment]
Ah, it works! I also had to change font.offset.y from -7 to -1 as it looked
really terrible. Maybe it is different with the default Linux monospace
font.

**Comment by jwilm** [comment]
Glad that fixed it! Closing in favor of #40.
